fontspec-code-keyval.dtx

```diff
@@ -795,6 +801,23 @@
 %</LU>
 %    \end{macrocode}
 %
+% For other potentially font specific variation axes, there is a raw setter available:
+%    \begin{macrocode}
+\@@_keys_define_code:nnn {fontspec-opentype} {RawAxis}
+  {
+    \prop_gput_from_keyval:Nn \g_@@_rawvariations_prop {#1}
+  }
+%    \end{macrocode}
+%
+% \subsubsection{Variation instances}
+%
+%    \begin{macrocode}
+\@@_keys_define_code:nnn {fontspec-opentype} {Instance}
```

Comment:
Is "Instance" specific enough or would "VariationInstance" be better? (Not sure, just thinking out loud). WDYT?

Reply:
I don't really have an opinion on that. Personally I prefer Instance, but that's a purely subjective opinion.